charts/celeborn/README.md

```diff
@@ -17,19 +17,20 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Helm Chart for Apache Celeborn
+# celeborn
 
-[Apache Celeborn](https://celeborn.apache.org) is an intermediate data service for Big Data compute engines (i.e. ETL, OLAP and Streaming engines) to boost performance, stability, and flexibility. Intermediate data typically include shuffle and spilled data.
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.4.0](https://img.shields.io/badge/AppVersion-0.4.0-informational?style=flat-square)
 
+Celeborn is dedicated to improving the efficiency and elasticity of different map-reduce engines. Celeborn provides an elastic and high efficient management service for shuffle data.
```

Comment:
Better keep this, and the content here should be consistent with the content on the Celeborn website see [Apache Celeborn](https://celeborn.apache.org/)

Reply:
> Better keep this, and the content here should be consistent with the content on the Celeborn website see [Apache Celeborn](https://celeborn.apache.org/)

Updated.